datums_warehouse/warehouse.py

```diff
@@ -0,0 +1,50 @@
+from datums_warehouse.source import KrakenSource
+from datums_warehouse.storage import Storage
+
+
+def make_storage(directory):
+    return Storage(directory)
+
+
+# noinspection PyShadowingBuiltins
+def make_source(type, **kwargs):
+    if type == 'Kraken':
+        return KrakenSource(**kwargs)
+
+    raise NotImplementedError(type)
```

Comment:
Maybe the pair should also be a config property of the packet like the interval and the storage and source are handed those parameters. The storage would build its path from it and the source would receive it as ctr parameter. This would make the setup more consistent.